__tests__/app.test.js

```diff
@@ -10,10 +10,9 @@ beforeEach(() => {
 
 afterAll(() => {
     return db.end();
-    // return seed(testData).then(() => db.end());
 });
 
-describe("GET /api/topics", () => {
+describe("GETs", () => {
```

Comment:
I think it was better how you had it before with the endpoint in the describe block

Reply:
Thanks a lot. I will do that.